src/includes/getting-started-config/javascript.nextjs.mdx

```diff
@@ -12,6 +12,18 @@ You'll be prompted to log in to Sentry. The wizard will then automatically add t
 
 See [manual configuration](/platforms/javascript/guides/nextjs/manual-setup/) for examples of the files created by the wizard. If any of these files already exists, it will be created with a leading underscore and you will need to merge it with the existing file manually.
 
-To complete your configuration, add [options](/platforms/javascript/configuration/) to your two `Sentry.init()` calls (in `sentry.client.config.js` and `sentry.server.config.js`, respectively). In those two files you can also set context data - data about the [user](/platforms/javascript/enriching-events/identify-user/), for example, or [tags](/platforms/javascript/enriching-events/tags/), or even [arbitrary data](/platforms/javascript/enriching-events/context/) - which will be added to every event sent to Sentry.
+To complete your configuration, add [options](/platforms/javascript/configuration/) to your two `Sentry.init()` calls (in `sentry.client.config.js` and `sentry.server.config.js`, respectively).
+In those two files you can also set context data - data about the [user](/platforms/javascript/enriching-events/identify-user/), for example, or [tags](/platforms/javascript/enriching-events/tags/), or even [arbitrary data](/platforms/javascript/enriching-events/context/) - which will be added to every event sent to Sentry.
```

Comment:
Unless you've gotten specific feedback to add these two examples of context, I think they're too much detail on the page, so edit to:
```suggestion
In those two files you can also set [context data](/platforms/javascript/enriching-events/) which will be added to every event sent to Sentry.
```

Reply:
That was me, and I did it because the Context page isn't about adding context in general, it's about adding stuff to the `contexts` field of an event. I agree that the sentence is kind of wordy, but I didn't know how else to not have the other two things you'd reasonably do in those files (set user and set tags) be totally missed. Open to suggestions.

(In the log run, my suggestion would be for that first context page to be about context data in general, demoting the current page to be a sibling of `setUser`, `setTags`, etc, rather than the context landing page. That's obv out of the scope of this PR, though.)